src/layout/MasterLayout.cpp

```diff
@@ -1110,15 +1117,17 @@ std::any CHyprMasterLayout::layoutMessage(SLayoutMessageHeader header, std::stri
         if (!PWINDOW)
             return 0;
 
-        const auto PNEXTWINDOW = getNextWindow(PWINDOW, true);
+        const bool noLoopAround = vars.size() >= 2 && vars[1] == "noLoopAround";
```

Comment:
noLoopAround is excessively verbose I'd say, why not just `noloop`?

also const vars in CAPS pls

Reply:
Yeah, way better. It was like that at the beginning but I felt it was more descriptive like that.

Would it be better to add a new layout command named `cyclenextnoloop` rather than making this a param for `cyclenext`?

Thanks for the feedback!